app/controllers/bodies_controller.rb

```diff
@@ -1,4 +1,7 @@
 class BodiesController < ApplicationController
+	def index
```

Comment:
Have a look at your indentation, looks like you are using tab and everyone else is using spaces. I think this is a setting in your IDE you can set. In sublime text we have the option to mark `Indent using spaces`